package.json

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
This file is required and used by Node as sort of metadata. It also keeps track of node modules if you choose to use them